src/main/java/kevesse_kokanyolo_kod/room/Room.java

```diff
@@ -92,9 +97,13 @@ public void popItem(Player player) {
         if (items.isEmpty()) {
             SkeletonMenu.endCall("Nincs több tárgy a szobában.");
             return;
+        } else if (lastCleaning.doesCleaningDenyItemPickup()) {
+            SkeletonMenu.endCall("Túl rég volt takarítás a szobában így azok ragacsosak lettek");
```

Comment:
A magyar nyelv szép... Ha még értelme is lenne a mondatnak.

Reply:
Szerintem az else maradhat, tekintve, hogy ha az első ha nem jó akkor a második if ág és ha egyikse akkor a metódus vége (A következő kommented okán a 103-as soron